src/components/People.tsx

```diff
@@ -0,0 +1,131 @@
+    getPeople()
+      .then(setPeople)
+      .catch(() => setErrorMessage('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {errorMessage && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {errorMessage}
+            </p>
+          )}
+
+          {people && !people.length && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {!errorMessage && people && people.length > 0 && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people &&
+                  people.map(person => {
+                    const moth = people.find(
+                      pers => pers.name === person.motherName,
+                    );
+
+                    const fath = people.find(
+                      pers => pers.name === person.fatherName,
+                    );
+
+                    const normalizedPerson = { ...person, moth, fath };
+
+                    const {
+                      name,
+                      sex,
+                      born,
+                      died,
+                      motherName,
+                      fatherName,
+                      slug,
+                    } = normalizedPerson;
+
+                    return (
+                      <tr
+                        data-cy="person"
+                        key={slug}
+                        className={classNames({
+                          'has-background-warning': slug === personSlug,
+                        })}
+                      >
+                        <td>
+                          <Link
+                            to={`../people/${slug}`}
+                            className={classNames({
+                              'has-text-danger': sex === 'f',
+                            })}
+                          >
+                            {name}
+                          </Link>
+                        </td>
+
+                        <td>{sex}</td>
+                        <td>{born}</td>
+                        <td>{died}</td>
+                        <td>
+                          {motherName ? (
+                            <PersonLink
+                              person={normalizedPerson.moth}
+                              personName={motherName}
+                            />
```

Comment:
```suggestion
                            <PersonLink person={mother}  />
```
You have a person's name in the person object...